src/hooks/usePokemons.tsx

```diff
@@ -48,11 +48,15 @@ const usePokemons = () => {
     fetchPokemons(offset);
   }, [offset]);
 
-  const loadMorePokemons = () => {
+  const loadPrevious = () => {
```

Comment:
Currently, there is no explicit check within the functions to ensure that the offset value remains within valid boundaries (e.g., ensuring the offset doesn't become negative). This could be something to add if it isn't handled elsewhere in the cod

Reply:
Since the Previous button blocks when your offset is zero, it will never go below zero